src/Uno.UI/UI/Xaml/RoutedEvent.cs

```diff
@@ -26,14 +22,34 @@ public RoutedEvent([CallerMemberName] string name = null)
 			IsGestureEvent = flag.IsGestureEvent();
 		}
 
+		[Pure]
 		internal string Name { get; }
 
+		[Pure]
 		internal RoutedEventFlag Flag { get; }
 
+		/// <summary>
+		/// Determines if this event is bubbled to the parent, not matter is a parent is subscribed
+		/// to the handler with the flag handledEventToo or not.
+		/// </summary>
+		/// <remarks>
+		/// For some events like PointerEvent, we always needs to get the full events sequence to maintain the
+		/// internal state, so we always needs the handled events too.
+		/// This flag avoids the complex update of the SubscribedToHandledEventsToo property coercing and inheritance
+		/// for those kind of well-known events.
+		/// </remarks>
+		[Pure]
+		internal bool IsAlwaysBubbled => IsPointerEvent;
+
+		[Pure]
```

Comment:
Why adding this now ? What checks it ?

Reply:
At the beginning I add it only on the `IsAlwaysBubbled` as this flag is used in code that might run quite frequently, I wanted to hint the compilator. But I agree that I'm not sure if the compilator use it, nor if it's needed on auto-properties.

BTW: ReSharper checks it.